src/Utilities/CMakeLists.txt

```diff
@@ -88,9 +88,16 @@ target_link_libraries(
   Boost::boost
   Brigand
   ErrorHandling
-  Libxsmm
   )
 
+if (NOT ENABLE_SPECTRE_DEBUG)
```

Comment:
Maybe add a small comment here saying "libxsmm is disabled in debug mode, so only link it in release" just so future people know why this was done

Reply:
done